src/main/java/com/calculator/calculator/CalculatorApp.java

```diff
@@ -18,6 +18,9 @@ public class CalculatorApp extends Application {
 
     public static void main(String[] args) {
         launch(args);
+        //ivan comment
+        // comment + 12
+        //next OK
```

Comment:
Ivan can you do something with this 12 please ?

Reply:
Hello, I think everything is fine